OpenRA.Mods.Common/Traits/Multipliers/ProductionCostMultiplier.cs

```diff
@@ -21,15 +21,15 @@ public class ProductionCostMultiplierInfo : TraitInfo<ProductionCostMultiplier>,
 		[Desc("Percentage modifier to apply.")]
 		public readonly int Multiplier = 100;
 
-		[Desc("Only apply this cost change if owner has these prerequisites.")]
+		[Desc("Only apply this cost change if the owner has these prerequisites.")]
 		public readonly string[] Prerequisites = { };
 
-		[Desc("Queues that this cost will apply.")]
-		public readonly HashSet<string> Queue = new HashSet<string>();
+		[Desc("Production queues that this cost will apply to.")]
+		public readonly HashSet<string> Queues = new HashSet<string>();
```

Comment:
Could you change those for ProductionTimeMultiplier too for consistency?

Reply:
Okay.

Reply:
I was referring to Queue -> Queues and the other desc fix too.